executors/cpp/main.cpp

```diff
@@ -109,11 +111,19 @@ int main(int argc, const char** argv)
       // else if (test_type == "number_fmt") {
       //   outputLine = test_numfmt(json_input);
       //   // }
-      // else if (test_type == "language_display_name") {
-      //   outputLine = test_langnames(json_input);
-      // }
-else {
-        std::string outputLine = "# BAD TEST " + test_type;
+      else if (test_type == "lang_names" ||
```

Comment:
We should really have only one value in this key (b/c it behaves like an enum) to represent this one component type.

Reply:
Agreed.